tests/local/etl/test_transformer.py

```diff
@@ -86,6 +90,53 @@ def test_not_consume_inputs(self):
         self.assertIs(self.df, result["TestTransformer"])
         self.assertIs(self.df, result["df"])
 
+    def test_non_consume_in_etl_context(self):
```

Comment:
Burde denne test ikke ligge i test_transformer_nc.py filen?
Og burde filerne måske bare samles?

Reply:
Good idea. I have merged the two tests!